go.mod

```diff
@@ -4,6 +4,7 @@ go 1.14
 
 replace (
 	github.com/Azure/go-autorest => github.com/Azure/go-autorest v14.2.0+incompatible
+	github.com/aws/aws-sdk-go => github.com/aws/aws-sdk-go master
```

Comment:
Do we need to use master branch ?
if it's necessary please define this replace in hack/go.mod.default

Reply:
The required version to build it correctly is later than v1.30.x. I'll add v1.33.21 to hack/go.mod.default

Reply:
updated! thanks @kpango